### PR TITLE
feat: docker fix (works with latest repo structure)

### PR DESCRIPTION
**Pull request by SwenSchaeferjohann**
- extracted from https://github.com/Lightprotocol/light-protocol/pull/327 into dedicated PR for clarity
- adds a working script for relayer deployment on digitalocean

### CODE REVIEW

**Comment by SwenSchaeferjohann** [comment]
preferably merge directly after #327

File: scripts/deployRelayer.sh [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+pnpm install
+cd ..
+
+
+# build docker image and deploy to digitalocean
+docker buildx create --name mybuilder
+docker buildx use mybuilder
+docker run --privileged --rm tonistiigi/binfmt --install all
+docker buildx build --platform linux/amd64 -t relayer-app:latest . --load
+docker tag relayer-app:latest registry.digitalocean.com/v3-relayer/relayer-app:latest
```
Comment (ananas-block) [inline scripts/deployRelayer.sh:94]: Two ideas how to test this script:

1. add a test flag which let's you skip line 94-96. This way you can build docker but not deploy it.
2. separate the script into two, the first builds docker, the second uses the first to build docker and deploys it. The test would only test the first script.

**Review by ananas-block — CHANGES_REQUESTED** [review]
I think it makes sense to add a test, otherwise looks good, to me.

File: scripts/buildDocker.sh
Comment (ananas-block) [inline scripts/buildDocker.sh:1]: What about naming these files ```buildDockerRelayer.sh```, ```deployDockerRelayer.sh```, ```testDockerRelayer.sh```?

**Review by ananas-block — APPROVED** [review]

**Pull request merged**